tests/Smoke/StdinTest.php

```diff
@@ -75,7 +75,7 @@ public function testFixingStdin(): void
     private function unifyFooter(string $output): string
     {
         return preg_replace(
-            '/Checked all files in \d+\.\d+ seconds, \d+\.\d+ MB memory used/',
+            '/Checked \d+ of \d+ files that can be fixed in \d+\.\d+ seconds, \d+\.\d+ MB memory used/',
```

Comment:
```suggestion
            '/Found \d+ of \d+ files that can be fixed in \d+\.\d+ seconds, \d+\.\d+ MB memory used/',
```

I think this will fix the `StdinTest` :)